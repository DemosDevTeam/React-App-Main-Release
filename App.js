//For code ex. using firebase see https://github.com/davideast/firebase-react-native-sample/blob/master/index.ios.js
/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 * @flow
 */

import React from 'react';
import {
  Platform,
  StyleSheet,
  Text,
  View,
  Button,
  RectangleButton,
  Alert
} from 'react-native';

import Root from './router.js';
import Home from "./Screens/Home";
const firebase = require('firebase');

// Initialize Firebase
const firebaseConfig = {
  apiKey: "AIzaSyAAERSrXStyHsU889OZGfrCFe1E2Bit_xs",
  authDomain: "demos-5e3db.firebaseapp.com",
  databaseURL: "https://demos-5e3db.firebaseio.com",
  storageBucket: "demos-5e3db.appspot.com",
};
var firebaseApp = firebase.initializeApp(firebaseConfig);

export default class App extends React.Component {
  render() {
    return (
        <Root/>
    );
  }
}
export {firebaseApp};
