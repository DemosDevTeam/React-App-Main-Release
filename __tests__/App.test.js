import 'react-native'
import React from 'react'

import App from '../App'

import renderer from 'react-test-renderer'

describe('<App />', () => {
    test('it renders', () => {
        let wrapper = shallow(<App />)

        expect(wrapper).toMatchSnapshot()
    });
})