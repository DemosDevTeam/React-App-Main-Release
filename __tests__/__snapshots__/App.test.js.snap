// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> it renders 1`] = `<NavigationContainer />`;
